src/styles/blocks/sections/about.scss

```diff
@@ -0,0 +1,86 @@
+.about {
+  @include page-grid;
+
+  --columns: 1;
+
+  align-content: center;
+
+  &__titles {
+    grid-column: 1 / -1;
+
+    @include on-tablet {
+      grid-column: 1 / 3;
+    }
+
+    @include on-desktop {
+      grid-column: span 5;
+    }
+
+    &-main {
+      margin: 0;
+      padding: 0;
+    }
+
+    &-red {
+      color: $accent-color;
+      font-size: 12px;
+      font-weight: 700;
+      margin: 0;
+      padding: 0;
+    }
+  }
+
+  &__conteiner {
+    grid-column: span 1 / -1;
+
+    @include on-tablet {
+      grid-column: 4 / 7;
+    }
+
+    @include on-desktop {
+      grid-column: span 5;
+    }
+  }
+
+  &__place-schedule {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    row-gap: 16px;
+    justify-content: end;
+
+    &-time {
+      list-style-type: none;
+      font-weight: 700;
+      font-size: 16px;
+      margin: 0;
+      padding: 0;
+    }
+
+    &-days {
+      list-style-type: none;
+      font-size: 16px;
+      font-weight: 400;
+      margin: 0;
+      padding: 0;
+    }
+    &-closed-title {
+      font-weight: 700;
+      font-size: 16px;
+    }
+
+    &-closed-text {
+      font-size: 16px;
+      font-weight: 400;
+    }
+  }
+  &__place-closed-title {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__place-closed-text {
+    font-size: 16px;
+    font-weight: 400;
```

Comment:
```suggestion
```

Reply:
Thanks fore review! I've fixed all issues, made new commit and pressed 're-request review' button. Hope everything is correct now.